apps/laboratory/app/library/solana/page.tsx

```diff
@@ -20,7 +20,10 @@ const config = {
   adapters: [solanaWeb3JsAdapter],
   networks,
   metadata: ConstantsUtil.Metadata,
-  customWallets: ConstantsUtil.CustomWallets
+  customWallets: ConstantsUtil.CustomWallets,
+  registerWalletStandard: {
+    solana: true
+  }
```

Comment:
I think we could have only 
```suggestion
  registerWalletStandard: true
```
wdyt?

Reply:
I explicitly made it an object so it can be extendable / customizable with other networks if/when we add others